src/i18n/it.js

```diff
@@ -47,7 +47,9 @@ export default {
       password_no_user_info_error: "La password si basa sulle informazioni dell'utente.",
       password_strength_error: 'La password è troppo debole.',
       user_exists: "L'utente esiste già.",
-      username_exists: 'Il nome utente esiste già.'
+      username_exists: 'Il nome utente esiste già.',
+      social_signup_needs_terms_acception:
+        'Si prega di accettare i Termini di servizio di seguito per continuare.'
```

Comment:
English: "Please agree to the Terms of Service below to continue."

Reply:
```suggestion
        'Si prega di accettare i seguenti Termini di servizio per continuare.'
```